### PR TITLE
US104674 - Put headers into header columns

### PR DESCRIPTION
**Pull request by mtjandra**
* Changing data structure to allow for multiple "headers" within a "header column"
* This is to support the two header (firstName and lastName) within one column

### CODE REVIEW

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 1]
```diff
@@ -98,14 +102,14 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
...
-			_headers: {
+			_headerColumns: {
 				type: Array,
 				value: [
-					{ key: 'displayName', sortClass: 'first-name', canSort: false, localizationKey: 'displayName' },
-					{ key: 'activityName', sortClass: 'activity-name', canSort: false, localizationKey: 'activityName' },
-					{ key: 'courseName', sortClass: 'course-name', canSort: false, localizationKey: 'courseName' },
-					{ key: 'submissionDate', sortClass: 'completion-date', canSort: false, localizationKey: 'submissionDate' },
-					{ key: 'masterTeacher', canSort: false, localizationKey: 'masterTeacher' }
+					[{ key: 'displayName', sortClass: 'first-name', canSort: false, localizationKey: 'displayName' }],
```
Comment (mtjandra) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: I think i'll need to pull out the key from here as well. There should be a header columnKey separate from a columnKey

Comment (mtjandra) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Alright I have split them out

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 2]
```diff
@@ -157,21 +180,24 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
...
 				});
 
 				return Promise.resolve();
 			});
 	}
 
 	_sort(e) {
-		const header = this._headers.find(h => h.key === e.currentTarget.id);
+		const headers = [].concat.apply([], this._headerColumns.map(headerColumn => headerColumn.headers));
+		const header = headers.find(h => h.key === e.currentTarget.id);
```
Comment (svanherk) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: IE11 doesn't support `find` (looks like this was broken on master)

Comment (mtjandra) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Nice

**Pull request merged**